### DISCUSSION TITLE
en_PP-OCRv4_rec_infer 模型推理老是字典不对，有的图片又可以

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#12308] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubantu 
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

请尽量不要包含图片在问题中/Please try to not include the image in the issue

    args = utility.parse_args()
    args.image_dir = "/a.png"
    args.rec_model_dir = "./ppmodel/en_PP-OCRv4_rec_infer
    args.rec_char_dict_path = "./ppocr/utils/en_dict.txt" 
    main(args)
模型用的是哪个字典
/ppocr/postprocess/rec_postprocess.py", line 83, in <listcomp>
    self.character[text_id]
IndexError: list index out of range

[2024/03/28 17:16:38] ppocr INFO: list index out of range

**Comment by tink2123** [comment]
方便提供一下测试图片吗？
